site/reference/fn/eval/README.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "`eval`"
+linkTitle: "eval"
+type: docs
+description: >
+  Execute function on resources
+---
+
+<!--mdtogo:Short
+    Execute function on resources
+-->
+
+`eval` executes function on resources in a directory and writes the output to
+the local filesystem in-place. Functions are packaged as container images or
+binary executables.
+
+If the function fails (i.e. exits with non-zero status code), `eval` will
+abort and the local filesystem is left intact.
+
+Refer to the [Imperative Function Execution] for detailed overview.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```shell
+kpt fn eval [DIR|-] [flags]
+```
+
+#### Args
+
+```shell
+DIR|-:
+  Path to the local directory containing resources. Defaults to the current
+  working directory. Using '-' as the directory path will cause `eval` to
+  read resources from `stdin` and write the output to `stdout`.
+```
+
+#### Flags
+
+```shell
+--image:
+  Container image of the function to execute e.g. `gcr.io/kpt-fn/set-namespace:v0.1`
+
+--exec-path:
+  Path to the local executable binary to execute as a function.
+  
+--fn-config:
+  Path to the file containing `functionConfig` for the function.
```

Comment:
Should we clarify here that this config file will not be ignored if it's in the directory?

Reply:
I think we will have to figure out a better way to tell `how to ignore KRM files` for different workflows such as `live`, `packaging` and `hydration`. So not adding anything for now.